src/Parser/MarksmanshipHunter/Modules/Features/CastEfficiency.js

```diff
@@ -83,9 +81,8 @@ class CastEfficiency extends CoreCastEfficiency {
     {
       spell: SPELLS.TRUESHOT,
       category: CastEfficiency.SPELL_CATEGORIES.COOLDOWNS,
-      getCooldown: (haste, combatant) => {
-        const availableCasts = combatant.owner.modules.trueshot.tsAvailableCasts;
-        return (combatant.owner.fightDuration / 1000) / availableCasts;
+      getCooldown: (_, combatant) => {
+        return combatant.owner.modules.trueshot.reducedCooldownWithTraits;
```

Comment:
Thanks for the change - did you check it gives the same results? To the comment - if the lambda is just one return statement it can be shortened to 
```jsx
getCooldown: (_, combatant) => combatant.owner.modules.trueshot.reducedCooldownWithTraits,
```